googler

```diff
@@ -339,6 +403,17 @@ class GoogleParser(HTMLParser.HTMLParser):
 
         self.set_handlers_to('main')
 
+    ### Ignore list ###
+    IGNORE_LIST = [
+        # "People also ask"
+        # Sample query: VPN
+        # Screenshot: https://i.imgur.com/yfcsoQz.png
+        {
+            'tag': 'div',
```

Comment:
> the ignore list is only useful for things that are in the main scope — we can avoid dropping into other scopes in the first place.

On second thought, this may not be true, but I think it would be better to defer adding the scope test until we really need it (we can certainly recognize a `handler_scope` key in selectors), because ignore list entry tests are pretty lightweight, and an extra scope test for `main` that passes more often than not might not even be worth the saving.